src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/ManageContactInformationCallbackHandler.java

```diff
@@ -122,6 +127,52 @@ public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    private CallbackResponse validateExperts(CallbackParams callbackParams) {
+        String authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
+        CaseData caseData = callbackParams.getCaseData();
+        CaseData.CaseDataBuilder builder = caseData.toBuilder();
+        List<String> errors = new ArrayList<>();
+
+        if (!isAdmin(authToken)) {
```

Comment:
Minor: Can we add some comments in here around why this check exists and what no PartyID for users means the user has done in the event

Reply:
Added